Source/Core/Core/HW/Sram.h

```diff
@@ -42,6 +42,12 @@ using CardFlashId = std::array<u8, 12>;
 
 #pragma pack(push, 1)
 
+struct Rtc
+{
+  Common::BigEndianValue<u32> rtc;
+  u8& operator[](size_t offset) { return reinterpret_cast<u8*>(&rtc)[offset]; }
```

Comment:
This should have a const variant

Reply:
what do you mean?

Reply:
```cpp
const u8& operator[](size_t offset) const { return reinterpret_cast<const u8*>(&rtc)[offset]; }

Reply:
ah, i thought you were talking about the u32 or struct.
makes sense i guess.
shouldn't the sram struct be the same then? (it isn't/wasn't)

Reply:
the const also makes it not modifiable , does it not? :)

Reply:
I believe he means both const and non-const variants should exist.

Reply:
Yup

Reply:
mind me asking what the use is of having both the const and non-const variant? 
shouldn't the sram be the same?